### DISCUSSION TITLE
Never will have a encoder?

### DISCUSSION THREAD

**Discussion by marcussacana** [pleonex/xdelta-sharp#11] [General]
This project has a very big history, already have more than 7 years that exist and keep being maintened.
But even after 7 years, we not have a patch creator, why?

**Comment by pleonex** [comment]
Basically because I didn't sign any contract with anyone to complete or support the project and because I don't get paid for it.
I have plenty other things to do in many other projects and personal life. None of my other projects requires the encoder, they only need the decoder. If I need the encoder, I run manually xdelta3.
Also, implementing the encoder is more difficult, as it requires knowledge and techniques from compression algorithms and I am no expert there. So it would take me a lot of time to first learn, implement, optimize, etc... for something I don't need at the moment.
That's basically why.

I would be very happy to review and accept any contribution from anyone willing to implement an encoder here.

**Reply by marcussacana** [reply]
I see, sad but understandable. Thx for reply
